articles/active-directory-b2c/tenant-management-check-tenant-creation-permission.md

```diff
@@ -58,7 +58,7 @@ Before you create an Azure AD B2C tenant, make sure that you've the permission t
 
 1. Under **Manage**, select **User Settings**.
 
-1. Review your **Tenant Creation** setting. If the settings is set to **No**, then contact your administrator to assign the tenant creator role to you. The setting is greyed out if you're not an administrator in the tenant.
+1. Review your **Restrict non-admin users from creating tenants** setting. If the settings is set to **No**, then contact your administrator to assign the tenant creator role to you. The setting is greyed out if you're not an administrator in the tenant.
```

Comment:
```suggestion
1.  Under **Default user role permissions**, review your **Restrict non-admin users from creating tenants** setting. If the setting is set to **No**, then contact your administrator to assign the tenant creator role to you. The setting is greyed out if you're not an administrator in the tenant.
```

Reply:
@mattchannn 

Please the change I've requested. If it makes sense, commit it, then I will proceed to sign off the PR.